tests/translator/input/graphqlapi_resolver_function_with_lambda_datasource.yaml

```diff
@@ -0,0 +1,41 @@
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  SuperCoolAPI:
+    Type: AWS::Serverless::GraphQLApi
+    Properties:
+      Name: SomeApi
+      SchemaInline: |
+        type Mutation {
+          addTodo(id: ID!, name: String, description: String, priority: Int): Todo
+        }
+      XrayEnabled: true
+      Auth:
+        Type: AWS_IAM
+      Tags:
+        key1: value1
+        key2: value2
+      DataSources:
+        Lambda:
```

Comment:
Can we get a test with multiple lambda datasources and one with lambda and ddb datasource? Just to cover all grounds

Reply:
Sure, I'll add one with lambda and ddb datasources both. I don't really think multiple of just one type checks anything that the both combined wouldn't so I'll leave it at that